api/app/signals/apps/api/v1/fields/category.py

```diff
@@ -85,3 +85,51 @@ def get_object(self, view_name, view_args, view_kwargs):
         return self.get_queryset().get(
             parent__slug=view_kwargs['slug'],
             slug=view_kwargs['sub_slug'])
+
+
+class PrivateCategoryHyperlinkedIdentityField(serializers.HyperlinkedIdentityField):
```

Comment:
@tcoenen Kijk even goed naar de structuur in het weergeven van de URL's. Ik heb HAL aangehouden en zoals we ook bij een signal doen een paretn child achtige constructie gemaakt.